src/library/url-params.js

```diff
@@ -0,0 +1,17 @@
+import queryString from 'query-string'
+
+const ADD_EXTERNAL_REPORT = 'add_external_report'
+
+// This function will take search params from the current URL, parse them, check if there are any params
+// that are supported by Offering API, and it will append these query params to Offering API URL provided as an
+// argument. It ensures that apiUrl won't get broken, even if if already has some search params included.
+export const appendOfferingApiQueryParams = apiUrl => {
+  const searchParams = queryString.parse(window.location.search)
```

Comment:
I just added something very similar here:
https://github.com/concord-consortium/portal-pages/pull/100/files#diff-a68c46c5cae358f4592444dfad4c8dddR24
In that case I used the built in searchParams instead of bringing in the query-string package.
What do you think about using the built in searchParams for consistency instead? 
I think the query-string package has more features and probably works in IE, but I don't think we need either of these features.

Reply:
Sure, thanks. I didn't know about `URL` actually. Just pushed a commit that updates this code.

Reply:
OK, need to look at it again, it passes tests, but it doesn't work with relative URLs.